src/sentry/static/sentry/app/views/organizationStream/overview.jsx

```diff
@@ -125,13 +131,30 @@ const OrganizationStream = createReactClass({
         this._poller.disable();
       }
     }
-    if (prevProps.params.searchId != this.props.params.searchId) {
-      this.onSavedSearchChange();
-    } else if (
-      prevProps.location.search != this.props.location.search ||
-      !isEqual(prevProps.selection, this.props.selection)
+
+    // If project selections have changed we need to get new processing issues.
+    if (!isEqual(prevProps.selection.projects, this.props.selection.projects)) {
```

Comment:
Do we have any tests around behavior inside of these lifecycle methods?

Reply:
No. My next test is to add acceptance and unit tests for this view.